istioctl/cmd/sysexits.go

```diff
@@ -16,12 +16,14 @@ package cmd
 
 import "strings"
 
-// Values should use sendmail-style values as in <sysexits.h>
+// Values should try to use sendmail-style values as in <sysexits.h>
 // See e.g. https://man.openbsd.org/sysexits.3
 // or `less /usr/includes/sysexits.h` if you're on Linux
 const (
 	ExitUnknownError   = 1 // for compatibility with existing exit code
 	ExitIncorrectUsage = 64
+	// below here are non-zero exit codes that don't indicate an error with istioctl itself
+	ExitAnalyzeFoundIssues = 79 // istioctl analyze found issues, for CI/CD
```

Comment:
Should we reserve a range for "errors that mean something went wrong, but we successfully finished executing anyway?" e.g. 100+

Reply:
Yes, but it's trickier than I thought at first. There's this page: https://www.tldp.org/LDP/abs/html/exitcodes.html#EXITCODESREF which defines some reserved ones, and the referenced sysexits.h which defines from 64 to 78. So I think 79-125 is the "usable" range here. I can add a comment explaining this a bit further.